app/services/resource_service.rb

```diff
@@ -53,7 +53,9 @@ def create_or_recreate(old_record, external_resource)
 
   def unchanged_attributes?(record)
     return false if record.blank?
+    return @resource.compareable_attributes == record.compareable_attributes if record.respond_to?(:compareable_attributes)
```

Comment:
was ist wenn hier `true` ist und was, wenn `false`.
wo wird `unchanged_attributes?` verwendet. hat das zur folge, dass etwas neu importiert wird oder nicht? heißt neu imporitert dann "delete+create"?

Reply:
das wird bisher schon in der create Methode vom ResourceService verwendet. abhängig dessen wird entweder der alte Datensatz zurückgegeben, oder der alte gelöscht und ein neuer erstellt
<img width="610" alt="Bildschirmfoto 2021-02-16 um 14 33 50" src="https://user-images.githubusercontent.com/90779/108069736-fdf99e80-7063-11eb-8cb7-34a57fde2d6f.png">

Reply:
das ist dann auch die Stelle für die Zukunft, an der wir das echte Update implementieren können/sollten/werden